README.md

```diff
@@ -53,8 +53,8 @@ Usage of vpc-network-config-analyzer:
         Prints the release version number
   -vpc string
         CRN of the VPC to analyze
-  -vpc-config string
-        Required. File path to input config
+  -vpc-config value
+        Required. File paths to input configs
```

Comment:
how can one understand from readme that `-vpc-config` can be used multiple times to provide multiple input config files in JSON format? 
same question for `-region` ?